### PR TITLE
feat: exit after play

### PR DESCRIPTION
**Pull request by codingCoffee**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [x] Feature
- [ ] Documentation update

## Checklist

- [ ] any anime playing
- [x] bumped version
---
- [ ] next, prev and replay work
- [ ] `-c` history and continue work
- [ ] `-d` downloads work
- [ ] `-s` syncplay works
- [ ] `-q` quality works
- [ ] `-v` vlc works
- [ ] `-e` select episode works
- [ ] `-S` select index works
- [ ] `-r` range selection works
- [ ] `--skip` ani-skip works
- [ ] `--skip-title` ani-skip title argument works
- [x] `--no-detach` no detach works
- [x] `--dub` and regular (sub) mode both work
- [ ] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [x] `-h` help info is up to date
- [x] Readme is up to date
- [x] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -266,6 +268,10 @@ play_episode() {
                 nohup "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 &
             else
                 "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode"
+                mpv_exitcode=$?
+                if [ "$exit_after_play" = 1 ]; then
+                  exit "$mpv_exitcode"
```
Comment (port19x) [inline ani-cli:273]:
```suggestion
                    exit "$mpv_exitcode"
```

**Comment by port19x** [comment]
Not too useful, but it's a small addition so we can add it anyway

**Comment by codingCoffee** [comment]
Resolved the review! Thanks!

File: ani-cli [hunk 2]
```diff
@@ -266,6 +268,10 @@ play_episode() {
                 nohup "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 &
             else
                 "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode"
+                mpv_exitcode=$?
+                if [ "$exit_after_play" = 1 ]; then
+                    exit "$mpv_exitcode"
+                fi
```
Comment (Derisis13) [inline ani-cli:274]: This should be wrapped into `[ "$exit_after_play" = 1 ] && exit "$mpv_exitcode"`.

Comment (codingCoffee) [reply ani-cli]: Fixed the problem!

**Review by port19x — APPROVED** [review]

**Pull request merged**